File: C++/First Steps in Coding - Basics of C++/Conditional Statements - Advanced/Exercises/Operations Between Numbers.cpp
#include <iostream>
#include <cstdlib>
#include <iomanip>
#include <string>
using namespace std;
int main() {
	int num1, num2;
	string op;
	cin >> num1 >> num2 >> op;
	if (op == "+") {
		cout << num1 << " " << op << " " << num2 << " = " << num1 + num2 << " - ";
		if ((num1 + num2) % 2 == 0) {
			cout << "even";
		}
		else {
			cout << "odd";
		}
	}
	else if (op == "-") {
		cout << num1 << " " << op << " " << num2 << " = " << num1 - num2 << " - ";
		if ((num1 - num2) % 2 == 0) {
			cout << "even";
		}
		else {
			cout << "odd";
		}
	}
	else if (op == "*") {
		cout << num1 << " " << op << " " << num2 << " = " << num1 * num2 << " - ";
		if ((num1 * num2) % 2 == 0) {
			cout << "even";
		}
		else {
			cout << "odd";
		}
	}
	else if (op == "/") {
		if (num2 == 0) {
			cout << "Cannot divide " << num1 << " by zero";
		}
		else {
			cout << fixed << setprecision(2) << num1 << " " << op << " " << num2 << " = " << (double)num1 / num2;
		}
	}
	else if (op == "%") {
		if (num2 == 0) {
			cout << "Cannot divide " << num1 << " by zero";
		}
		else {
			cout << num1 << " " << op << " " << num2 << " = " << num1 % num2;
		}
	}
	return 0;
}

File: Python/Basics/Conditional Statements Advanced/Exercise/OperationsBetweenNumbers.py
n1 = int(input())
n2 = int(input())
operator = input() # +, -, *, /, %
output = ""

if operator == "+" or operator == "-" or operator == "*":

    output = f"{n1} {operator} {n2} = {eval(f'{n1}{operator}{n2}')}"

    even = eval(f"{n1}{operator}{n2}") % 2

    if even == 0:
        output += " - even"
    else:
        output += " - odd"

elif operator == "/":
    if n2 == 0:
        print(f"Cannot divide {n1} by zero")
    else:
        output = f"{n1} {operator} {n2} = {eval(f'{n1}{operator}{n2}'):.2f}"

elif operator == "%":
    if n2 == 0:
        print(f"Cannot divide {n1} by zero")
    else:
        output = f"{n1} {operator} {n2} = {eval(f'{n1}{operator}{n2}')}"


print(output)